.bin/install-gitleaks-linux-x64.sh

```diff
@@ -2,7 +2,7 @@
 set -xeuo pipefail;
 
 version="8.12.0";
-releases_api="https://api.github.com/repos/zricethezav/gitleaks/releases/latest";
```

Comment:
This is failing because `latest` is no longer `8.12.0` but is now `8.13.0`; why not upgrade instead?

Reply:
No reason, just thought it would've been more appropriate vs upgrading. Looking at the changelog though there isn't anything breaking so I'll go ahead and update